utilities/pipelines/resourceRemoval/helper/Invoke-ResourceRemoval.ps1

```diff
@@ -77,6 +77,16 @@ function Invoke-ResourceRemoval {
             $roleAssignmentsOnScope | Where-Object { $_.RoleAssignmentId -eq $ResourceId } | Remove-AzRoleAssignment
             break
         }
+        'Microsoft.ManagedServices/registrationAssignments' {
```

Comment:
Are we sure all this custom logic is needed and it is not enough to just enable the removal by default?

Reply:
It's a result of the corresponding issue #644 - hence I expected it to be required. I'll go ahead and run it once and force it to try and remove the definition

Reply:
True story. Just testedit in main and it 'seems' to work. Undid the changes now and re-run from this branch.